docs/PfSense.md

```diff
@@ -0,0 +1,20 @@
+In order to boot PfSense via PXE and use unattended install,
+you need the following:
+
+1. Extracted PfSense installation ISO exported via NFS.
+
+2. Relevant entry in DHCP server config:
+    `option root-path "/srv/tftp/images/pfsense";`
+   /srv/tftp/images/pfsense is the path to the extracted ISO.
+
+3. Relevant entries in PXELINUX config:
+```
+label PfSense
+ menu label PfSense
+ pxe images/pfsense/boot/pxeboot
+```
+
+The path `images/pfsense/boot/pxeboot` is relative to TFTP root directory.
+It's a path to the PXE loader in the extracted PfSense installation image.
+
+Unfortunately, there's no mechanism for unattended installation of PfSense.
```

Comment:
@pkubaj any reference about that ?

Reply:
There's just no reference to unattended installation in the docs or in the official forum. It's possible though too install with Foreman.
https://doc.pfsense.org/index.php/Installing_pfSense - nothing here
Other people also ask about it:
https://forum.pfsense.org/index.php?topic=87212.0
https://forum.pfsense.org/index.php?topic=61112.0
https://forum.pfsense.org/index.php?topic=66024.0

Also, OPNSense, which forked from PfSense, also doesn't support it.

Reply:
@pkubaj please add this information as note to documentation. It would be very helpful for future developers.

Reply:
Done.